README.md

```diff
@@ -266,62 +266,76 @@ compression_ctrl.export_model("compressed_model.pb", save_format="frozen_graph")
+| [TensorFlow Model Quantization](https://github.com/openvinotoolkit/openvino_notebooks/tree/main/notebooks/301-tensorflow-training-openvino)                                                                                                                                                                                                                                                              |                               Post-Training Quantization                                | Tensorflow |        Image Classification         |
+| [Migrating from POT to NNCF](https://github.com/openvinotoolkit/openvino_notebooks/tree/main/notebooks/111-yolov5-quantization-migration)<br>[![Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/openvinotoolkit/openvino_notebooks/blob/main/notebooks/111-yolov5-quantization-migration/111-yolov5-quantization-migration.ipynb)             |                               Post-Training Quantization                                |  OpenVINO  |          Object detection           |
+| [Quantization with Accuracy Control](https://github.com/openvinotoolkit/openvino_notebooks/tree/main/notebooks/122-quantizing-model-with-accuracy-control)                                                                                                                                                                                                                                               |                    Post-Training Quantization with Accuracy Control                     |  OpenVINO  | Speech-to-Text,<br>Object Detection |
+| [TensorFlow Training-Time Compression](https://github.com/openvinotoolkit/openvino_notebooks/tree/main/notebooks/301-tensorflow-training-openvino)                                                                                                                                                                                                                                                       |                                Training-Time Compression                                | Tensorflow |        Image Classification         |
+| [Joint Pruning, Quantization and Distillation for BERT](https://github.com/openvinotoolkit/openvino_notebooks/blob/main/notebooks/116-sparsity-optimization)                                                                                                                                                                                                                                             |                      Joint Pruning, Quantization and Distillation                       |  OpenVINO  |                 NLP                 |
+
+Please expand the table below to see the list of notebooks demonstrating OpenVINO conversion and inference together with NNCF compression for models from various domains.
+
```

Comment:
What do you think about moving the table below to `examples/openvino/README.md` and adding link here?

Reply:
I personally think that it's import to highlight this in the main readme, because the list contains many trend models and it would be a pity to hide this information. Maybe in the future we could move some outdated models to a separate section.

Reply:
Unfortunately, this list is not completely aligned with NNCF-enabled OV notebooks. Also the `Backend` column only contains one value. I don't think it's needed. In my opinion, a drop-down list with different domains is more convenient for easier navigation.

Reply:
Thanks for your suggestions!

#### Comment 1
> Unfortunately, this list is not completely aligned with NNCF-enabled OV notebooks.

Have I missed some notebooks? Could you please elaborate more on this.

#### Comment 2
> Also the Backend column only contains one value. I don't think it's needed.

For now it indeed does. Here's how it looks without Backend column: [preview](https://github.com/openvinotoolkit/nncf/blob/70c82e200a0079fb503c7388afb12a3edc12a788/README.md#demos-tutorials-and-samples). I personally would like to keep it because:
- If we remove it for one table, it won't be aligned with other tables, harming the overall look
- In the future there may be other values besides OpenVINO, in this case we'll need to bring the column back

#### Comment 3
> In my opinion, a drop-down list with different domains is more convenient for easier navigation.

Here's how it looks with two drop down sections for CV and NLP: [preview](https://github.com/openvinotoolkit/nncf/blob/623e0d594216f9fe8e1644d8fc3f311740f7d0e5/README.md#demos-tutorials-and-samples). And with Domain column removed: [preview](https://github.com/openvinotoolkit/nncf/blob/808ddd32b6489ec7d2acd13490ad2b154bfe947e/README.md#demos-tutorials-and-samples). I'd keep it as is because:
- I'm not really into hiding these. As I stated before, when these are hidden I feel like people won't bother expanding it and will miss it. Also, page search doesn't work in this case. Moreover, our page may not be indexed by search engine based on these model keywords. I have not tested it though.
- I've split it into two sections: CV and NLP because a more fine grained differentiation would be too much. However, if we keep the Domain column, I don't see a point in this split into two sections as the information is still there. On the other hand, if we remove the Domain column, then it again does not look aligned with other tables, and moreover it gets hard to understand what every notebook actually does, i.e. CV and NLP labels poorly describe the contents.

All in all, I'd keep it as is. But maybe a third opinion would be valuable on this.

cc @MaximProshin